welcome/src/main/kotlin/edu/artic/welcome/WelcomeFragment.kt

```diff
@@ -52,25 +51,21 @@ class WelcomeFragment : BaseViewModelFragment<WelcomeViewModel>() {
             }
         }
 
-        context?.let {
-            /* Build tour summary list*/
-            val layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
-            tourSummaryRecyclerView.layoutManager = layoutManager
+        /* Build tour summary list*/
+        val layoutManager = LinearLayoutManager(activity, LinearLayoutManager.HORIZONTAL, false)
+        tourSummaryRecyclerView.layoutManager = layoutManager
 
-            val decoration = DividerItemDecoration(context, DividerItemDecoration.HORIZONTAL)
-            decoration.setDrawable(ContextCompat.getDrawable(it, R.drawable.space_decorator)!!)
-            tourSummaryRecyclerView.addItemDecoration(decoration)
+        val decoration = DividerItemDecoration(view.context, DividerItemDecoration.HORIZONTAL)
+        decoration.setDrawable(ContextCompat.getDrawable(view.context, R.drawable.space_decorator)!!)
```

Comment:
`!!` not ideal. :( maybe we can extract into an extension method `getDrawableCompat` that explicitly unwraps it from the method call instead.